native/cocos/platform/BasePlatform.cpp

```diff
@@ -66,4 +74,14 @@ BasePlatform* BasePlatform::getPlatform() {
 #endif
     return &platform;
 }
-} // namespace cc
\ No newline at end of file
+
+BasePlatform* BasePlatform::getPlatform() {
```

Comment:
getPlatform -> getInstance the same logic as other module.

Reply:
getPlatform is a static member method, and getInterface is an ordinary member method. Two are not the same. Most of the time, the interface is obtained through the engine, reducing the direct dependence on the platform

Reply:
In cocos, always use getInstance to get singleton object.

Reply:
Don't have to use this method? Wouldn't it be better to use more direct semantics? Or change to getCurrentPlatform?